src/core/utils/helpers/url.ts

```diff
@@ -34,6 +34,12 @@ export const setQueryParametersInUrl = (parameters: {
   window.history.replaceState(null, "", processedUrl);
 };
 
+export const removeUrlQueryParam = (param: string) => {
+  const processedUrl = new URL(getCurrentLocation());
+  processedUrl.searchParams.delete(param);
+  window.history.replaceState(null, "", processedUrl);
+};
+
```

Comment:
The is already a `removeQueryParametersFromUrl()` in the same file. Please consolidate the two functions.